packages/admin/blocks-admin/src/blocks/factories/createCompositeBlock.tsx

```diff
@@ -44,6 +44,7 @@ interface CreateCompositeBlockOptionsBase {
     category?: BlockCategory | CustomBlockCategory;
     adminLayout?: "stacked";
     blocks: Record<string, BlockConfiguration>;
+    visibleOrderedBlocksForState?: (state: unknown) => string[];
```

Comment:
I'm not convinced about the name of this method. Maybe it would be good to include adminComponents somehow, because this is only relevant for the Admin.

We should also add a JSDoc comment to explain what this is used for.

Reply:
I guess it's the best we can come up with 😅 With you JSDoc comment it's fine 👍🏼